### DISCUSSION TITLE
What steps do I have to do again?

### DISCUSSION THREAD

**Discussion by CaneRandagio1983** [vzhd1701/evernote-backup#41] [Q&A]
I made my first backup following steps 1, 2, 3 of [README](https://github.com/vzhd1701/evernote-backup#usage)
But next time I have to do next backup, do I have to proceed with all 3 steps again? 
or can I start from the second step?

Thanks

**Comment by vzhd1701** [comment]
Step 1 must be run once when you initialize database. On daily backup just run Step 2.
